packages/core/stories/tokens/typography/Typography.stories.mdx

```diff
@@ -90,130 +83,131 @@ Using these fonts for your print and digital assets is a subtle but compelling w
   }
 </div>
 
-## Languages by Script
+## Language support
 
-The following languages are officially supported by [Mass.gov](https://www.mass.gov). If your project needs to support an addtional language, please [submit a feature request](https://github.com/massgov/mayflower/issues/new/choose) on Github.
+The following languages are officially supported by [Mass.gov](https://www.mass.gov):
 
-<!--
-@todo look into font rendering issues below.
-The only fonts that are rendering with Noto are: Latin, Greek, Cyrillic, Khmer, Arabic and  Hebrew.
--->
+Arabic, Armenian, Chinese (Simplified), Filipino, French, German, Greek, Gujarati, Haitian Creole, Hebrew, Hindi, Italian, Japanese, Khmer, Korean, Lao, Malayalam, Persian, Polish, Portuguese, Russian, Spanish, Tamil, Telugu, Thai, Ukrainian, Urdu, and Vietnamese.
 
-### Left-to-right
+The foolowing do not yet have webfont support: Chinese (Simplified), Japanese, and Korean.
+
+If your project needs to support an addtional language, please [submit a feature request](https://github.com/massgov/mayflower/issues/new/choose) on Github.
+
+### Languages by script
 
 ---
 
+#### Left-to-right
+
 #### Armenian script
 
-<!-- @todo investigate why Note font is not rendereing here -->
 <p className="lang-label">Armenian: </p>
-<p lang="hy" className="font-specimen">Բարի գալուստ Մասաչուսեթս</p>
+<p lang="hy" className="font-specimen">Ու նայիրեան աղջիկների հեզաճկուն պա՛րն եմ սիրում։</p>
```

Comment:
what is this text? can we revert it to the original text -- which is the title of the mass.gov homepage?